fastlane/lib/fastlane/actions/pod_push.rb

```diff
@@ -2,11 +2,15 @@ module Fastlane
   module Actions
     class PodPushAction < Action
       def self.run(params)
+        command = ''
+
+        command << "bundle exec" if params[:use_bundle_exec] && shell_out_should_use_bundle_exec?
```

Comment:
Mh, are you sure this works? Looks like there is no `&&` inbetween the commands, so this should crash? Did it work when you tested it locally?

Reply:
Hello @KrauseFx 

Sorry I must confess I do not have a pod project to test again, I will wait for a prod release and test.
As for your remark I think I do not need && since i want to run `bundle exec pod ...` instead of `pod ..` 

Still I got confused because `command` is a collection for the lib lint action and here a simple string. I think I am just missing a space after the word exec, do you agree ?